ports/libsimpleservo/src/api.rs

```diff
@@ -369,6 +371,9 @@ impl ServoGlue {
                     if let Err(e) = response_chan.send(true) {
                         warn!("Failed to send allow_navigation() response: {}", e);
                     };
+                    self.callbacks
+                        .host_callbacks
+                        .on_allow_navigation(_url.to_string());
```

Comment:
The java function "on_allow_navigation" need to return a boolean.

And instead of doing `if let Err(e) = response_chan.send(true) {` you should send the boolean value returned by the java function.

Reply:
Rename _url to url

Reply:
Done.